lazybytez/jojo-bot-website
# Add deployment for website

## Description
Add a deployment for the website.

## Related issue
There is no issue, as this is only relevant for us.

## Test Cases
 - Check that the staging site is reachable
 - Check that no secrets or internal URLs are leaked during deployment
